xml/admin_operating_services.xml

```diff
@@ -10,245 +10,204 @@
+      </para>
+     </listitem>
+    </varlistentry>
+   </variablelist>
    <para>
-    To start, stop or restart the <literal>osd.1</literal> service, run:
+    The following commands will help you list the unit files specific to the
+    node where you run them.
    </para>
-<screen>
-&prompt.root;systemctl start ceph-osd@1.service
-&prompt.root;systemctl stop ceph-osd@1.service
-&prompt.root;systemctl restart ceph-osd@1.service
-</screen>
    <para>
-    Commands for the other service types are analogous.
-   </para>
-  </sect2>
-
-  <sect2 xml:id="ceph-operating-services-finding-names">
-   <title>Identifying Individual Services</title>
-   <para>
-    You can find out the names/numbers of a particular type of service in
-    several ways. The following commands provide results for
-    <literal>ceph*</literal> services. You can run them on any node of the
-    &ceph; cluster.
-   </para>
-   <para>
-    To list all (even inactive) services of type <literal>ceph*</literal>, run:
+    To list all (even inactive) &ceph; services and targets on a node, run:
    </para>
 <screen>
-&prompt.root;systemctl list-units --all --type=service ceph*
+&prompt.root;systemctl list-units --all --type=service,target ceph*
 </screen>
    <para>
     To list only the inactive services, run:
    </para>
 <screen>
 &prompt.root;systemctl list-units --all --state=inactive --type=service ceph*
 </screen>
-   <para>
-    You can also use <command>salt</command> to query services across multiple
-    nodes:
-   </para>
+   <tip>
+    <title>Listing Services and Targets across Multiple Nodes</title>
+    <para>
+     You can also use <command>salt</command> to query services across multiple
+     nodes:
+    </para>
 <screen>
 &prompt.smaster;salt <replaceable>TARGET</replaceable> cmd.shell \
  "systemctl list-units --all --type=service ceph* | sed -e '/^$/,$ d'"
 </screen>
-   <para>
-    Query storage nodes only:
-   </para>
-<screen>
-&prompt.smaster;salt -I 'roles:storage' cmd.shell \
- 'systemctl list-units --all --type=service ceph*'
-</screen>
+   </tip>
   </sect2>
 
-  <sect2 xml:id="ceph-operating-services-status">
-   <title>Service Status</title>
+  <sect2 xml:id="ceph-operating-services-targets">
+   <title>Operating All Services on a Node</title>
    <para>
-    You can query &systemd; for the status of services. For example:
+    By using &ceph;'s &systemd; targets, you can simultaneously operate either
+    <emphasis>all</emphasis> services on a node, or all services that
+    <emphasis>belong to a cluster</emphasis> identified by its
+    <replaceable>FSID</replaceable>.
    </para>
-<screen>&prompt.root;systemctl status ceph-osd@1.service
-&prompt.root;systemctl status ceph-mon@<replaceable>HOSTNAME</replaceable>.service</screen>
    <para>
-    Replace <replaceable>HOSTNAME</replaceable> with the host name the daemon
-    is running on.
+    For example, to stop all &ceph; services on a node regardless to which
+    cluster they belong to, run:
```

Comment:
Mismatched numbers: the sentece starts talking about a singular then switches to talking about a plural.

Either:

"all ceph services on nodes regardless of which clusters they belong to"

or:

"all ceph services on a node, regardless of which cluster it belongs to"

Reply:
best would be "all ceph services on a node regardless of which clusters the services belong to"